python/rsgislib/classification/classaccuracymetrics.py

```diff
@@ -175,7 +175,10 @@ class name.
                           a JSON file (Default=None).
     :param out_csv_file: if specified the generated metrics and confusion matrix are written to
                          a CSV file (Default=None).
-
+    Example::
+    
+    import rsgislib.classification
+    rsgislib.classification.calc_acc_metrics_vecsamples("Sonoma_County_RF_classification_ref_samples.gpkg", "classification", "ref_samples", "RF_classes", "Sonoma_County_LCLUC_classification_2015.kea", img_cls_name_col='Classification', img_hist_col="Histogram", out_json_file="Sonoma_County_LCLUC_classification_2015_acc_metrics.json", out_csv_file=None)
```

Comment:
or would you prefer:

vector_file = "Sonoma_County_RF_classification_ref_samples.gpkg"
ref_col ="classification"
etc...

rsgislib.classification.calc_acc_metrics_vecsamples(vector_file, ref_col, ...)

Reply:
I'd probably specify the variables and then input them into the function as it makes it clearer what the input parameters are. 

Also, could you submit the pull request to the v5 branch and not the master? I'll reget the ones to the master as the code might be quite different for the v5 branch. Make sure you are checking out this branch when you edit.

Many thanks!